crates/services/consensus_module/poa/Cargo.toml

```diff
@@ -12,10 +12,12 @@ version = { workspace = true }
 [dependencies]
 anyhow = { workspace = true }
 async-trait = { workspace = true }
+aws-sdk-kms = { version = "1.37.0", optional = true }
 fuel-core-chain-config = { workspace = true }
 fuel-core-services = { workspace = true }
 fuel-core-storage = { workspace = true }
 fuel-core-types = { workspace = true }
+k256 = { version = "0.13.3", features = ["ecdsa-core"] }
```

Comment:
Why do we need `ecdsa-core` and why we didn't use it before?

Reply:
It's needed for DER signature parsing. AWS returns signatures in DER format.